modules/conversion-center/classes/render/core-render.php

```diff
@@ -0,0 +1,217 @@
+							'alt'   => '',
+							'class' => 'e-link-in-bio__identity-image',
+							'src'   => esc_url( $identity_image_value['url'] ),
+						] );
+						?>
+						<img <?php echo esc_attr( $widget->get_render_attribute_string( 'icon-link' ) ); ?> />
+					<?php }; ?>
+				</figure>
+			<?php endif; ?>
+			<?php if ( $has_bio_heading || $has_bio_title || $has_bio_description ) : ?>
+				<div class="e-link-in-bio__content">
+					<?php if ( $has_bio_heading ) {
+						$widget->add_render_attribute( 'heading', 'class', 'e-link-in-bio__heading' );
+						$bio_heading_output = sprintf( '<%1$s %2$s>%3$s</%1$s>', Utils::validate_html_tag( $bio_heading_props_tag ), $widget->get_render_attribute_string( 'heading' ), $bio_heading_value );
+						Utils::print_unescaped_internal_string( $bio_heading_output );
+					} ?>
+					<?php if ( $has_bio_title ) {
+						$widget->add_render_attribute( 'title', 'class', 'e-link-in-bio__title' );
+						$bio_title_output = sprintf( '<%1$s %2$s>%3$s</%1$s>', Utils::validate_html_tag( $bio_title_props_tag ), $widget->get_render_attribute_string( 'title' ), $bio_title_value );
+						Utils::print_unescaped_internal_string( $bio_title_output );
+					} ?>
+					<?php if ( $has_bio_description ) {
+						$widget->add_render_attribute( 'description', 'class', 'e-link-in-bio__description' );
+						$bio_description_output = sprintf( '<p %1$s>%2$s</p>', $widget->get_render_attribute_string( 'description' ), $bio_description_value );
+						Utils::print_unescaped_internal_string( $bio_description_output );
+					} ?>
+				</div>
+			<?php endif; ?>
+			<?php if ( $has_icons ) : ?>
+				<div class="e-link-in-bio__icons">
+					<?php
+					foreach ( $icons_value as $key => $icon ) {
+						// Bail if no icon
+						if ( empty( $icon['icon_icon'] ) ) {
+							break;
+						}
+
+						// Check for link format based on type
+						$formatted_link = $icon['icon_url']['url'] ?? '';
+
+						// Ensure we clear the default link value if the matching type value is empty
+						switch ( $icon['icon_platform'] ) {
+							case 'Email':
+								$formatted_link = ! empty( $icon['icon_mail'] ) ? "mailto:{$icon['icon_mail']}" : '';
+								break;
+							case 'Telephone':
+								$formatted_link = ! empty( $icon['icon_number'] ) ? "tel:{$icon['icon_number']}" : '';
+								break;
+							case 'Telegram':
+								$formatted_link = ! empty( $icon['icon_number'] ) ? "https://telegram.me/{$icon['icon_number']}" : '';
+								break;
+							case 'Waze':
+								$formatted_link = ! empty( $icon['icon_number'] ) ? "https://www.waze.com/ul?ll={$icon['icon_number']}&navigate=yes" : '';
+								break;
+							case 'WhatsApp':
+								$formatted_link = ! empty( $icon['icon_number'] ) ? "https://wa.me/{$icon['icon_number']}" : '';
+								break;
+						}
+
+						// Bail if no link
+						if ( empty( $formatted_link ) ) {
+							break;
+						}
+
+						$widget->add_render_attribute( "icon-link-{$key}", [
```

Comment:
```suggestion
						$widget->add_render_attribute( 'icon-link-' . $key, [
```
We prefer this way :)

Reply:
@nicoladj77 please check this everywhere (if you haven't already)